packages/e2e/cypress/integration/ui/components/authenticator/sign-in-force-new-password/sign-in-force-new-password.steps.ts

```diff
@@ -10,7 +10,11 @@ And('I type in the phone number {string}', (phone: string) => {
 });
 
 And('I type in the password {string}', (password: string) => {
-  cy.findByPlaceholderText(/password/i).type(Cypress.env(password));
+  cy.findByPlaceholderText(/^password$/i).type(Cypress.env(password));
+});
+
+And('I confirm the password {string}', (password: string) => {
+  cy.findByPlaceholderText(/^confirm_password$/i).type(Cypress.env(password));
```

Comment:
These tests are skipped but I think the regex for the placeholder selector should be `/^confirm password$/i`.

Reply:
Good point! It should be. I'll have another PR following this one pretty quickly so I'll make the change there.